ZadanieTDD/payment_processor.py

```diff
@@ -0,0 +1,84 @@
+import time
+import logging
+from payment_gateway import PaymentGateway, TransactionResult, TransactionStatus
+from exceptions import PaymentException, NetworkException, RefundException
+
+# Konfiguracja loggera
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
`REQUIRED` I would put logger into the class constructor, to spy on it. Probably could need some abstraction layer.